### PR TITLE
PODS-1895 : H&S sch reg - Establisher - Company - create screen for 'does X have a Company CRN?'

### PR DESCRIPTION
**Pull request by abhijeetardale**
PODS-1895 : H&S sch reg - Establisher - Company - create screen for 'does X have a Company CRN?'

### CODE REVIEW

File: conf/establishersCompany.routes [hunk 1]
```diff
@@ -262,8 +264,10 @@ POST       /:index/company/what-you-will-need
...
 POST       /:index/company/:mode/what-you-will-need/:srn                                                    controllers.register.establishers.company.WhatYouWillNeedCompanyDetailsController.onSubmit(mode: Mode, srn: Option[String], index: Index)
 
-GET        /:index/company/company-number                                                                   controllers.register.establishers.company.HasCompanyNumberController.onPageLoad(mode: Mode = NormalMode, srn: Option[String] = None, index: Index)
-POST       /:index/company/company-number                                                                   controllers.register.establishers.company.HasCompanyNumberController.onSubmit(mode: Mode = NormalMode, srn: Option[String] = None, index: Index)
+GET        /:index/company/has-company-numbe                                                                controllers.register.establishers.company.HasCompanyNumberController.onPageLoad(mode: Mode = NormalMode, srn: Option[String] = None, index: Index)
```
Comment (grantrobson) [inline conf/establishersCompany.routes]: Missing "r"

File: conf/establishersCompany.routes [hunk 2]
```diff
@@ -262,8 +264,10 @@ POST       /:index/company/what-you-will-need
 GET        /:index/company/:mode/what-you-will-need/:srn                                                    controllers.register.establishers.company.WhatYouWillNeedCompanyDetailsController.onPageLoad(mode: Mode, srn: Option[String], index: Index)
 POST       /:index/company/:mode/what-you-will-need/:srn                                                    controllers.register.establishers.company.WhatYouWillNeedCompanyDetailsController.onSubmit(mode: Mode, srn: Option[String], index: Index)
 
-GET        /:index/company/company-number                                                                   controllers.register.establishers.company.HasCompanyNumberController.onPageLoad(mode: Mode = NormalMode, srn: Option[String] = None, index: Index)
```
Comment (grantrobson) [inline conf/establishersCompany.routes]: Do we need to get rid of the old URL?

Comment (abhijeetardale) [reply conf/establishersCompany.routes]: its not old, its new was newly added which got updated with prefix has-

File: conf/establishersCompany.routes (hunk 1 above)
Comment (abhijeetardale) [reply conf/establishersCompany.routes]: fixed

**Pull request merged**